src/utilities/visualization.js

```diff
@@ -0,0 +1,73 @@
+import { format } from 'd3-format'
+import { timeFormat, timeParse } from 'd3-time-format'
+
+export const formatDate = timeFormat('%b. %e')
```

Comment:
There's a format time and format number component in `/src/components/common/format` - maybe we can reuse them? I'm trying to keep all the date format stuff in one place because it lets us maintain a consistent style.